src/app.magic.ts

```diff
@@ -264,9 +264,12 @@ function shells(): [Path, string][] {
 
   const std = (shell: string) => `source <(tea --magic=${shell})  #docs.tea.xyz/magic`
 
+  const bash = 'test -z "$TEA_HAS_RUN" && source /dev/stdin <<<"$(tea --magic=bash) && export TEA_HAS_RUN=1  #docs.tea.xyz/magic'
```

Comment:
exporting `TEA_HAS_RUN` to all the user’s terminals is a little gross :/

dunno a good way around it.

notably I was pkging conda.org today and they only add to `.bash_profile`

Reply:
Maybe that's correct? It's possible non login shells inherit from login shells most of the time.